org.gnome.Music.json

```diff
@@ -0,0 +1,125 @@
+{
+    "app-id": "org.gnome.Music",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "gnome-music",
+    "rename-icon": "gnome-music",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Tracker1",
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--talk-name=com.intel.dleyna-server",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-music"
```

Comment:
`:ro` probably? Don't remember music writing anything.

Reply:
I don't think it does that, gnome music doesn't fetch online covert, nor export a `.m3u` playlist file.